vars/redhat-8.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Copyright 2015, Jean-Philippe Evrard <jean-philippe@evrard.me>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+keepalived_package_name: "keepalived"
+keepalived_service_name: "keepalived"
+keepalived_config_file_path: "/etc/keepalived/keepalived.conf"
+_keepalived_daemon_options_file_path: "/etc/sysconfig/keepalived"
+
+# Packages for Selinux
+keepalived_selinux_packages:
+  - python3-policycoreutils
+  - libselinux
+  - libselinux-devel
+  - checkpolicy
```

Comment:
`checkpolicy` is a dependency of python-policycoreutils, can you remove it, please?